### PR TITLE
Add get_bounds to Utilities

### PR DESCRIPTION
**Pull request by joehuchette**
Computes variable lower/upper bounds set on a MOI variable

### CODE REVIEW

**Comment by blegat** [comment]
What about `MOI.EqualTo`, if it's set we could return both bounds equal to the value of the set.

**Comment by joehuchette** [comment]
But it's invalid to have an Interval AND a GreaterThan, correct?

**Comment by odow** [comment]
> But it's invalid to have an Interval AND a GreaterThan, correct?

Yes. Here are the options:

 - Interval
 - LessThan
 - GreaterThan
 - EqualTo
 - LessThan AND GreaterThan
 - Semicontinuous
 - Semiinteger

File: src/Utilities/variables.jl [hunk 1]
```diff
@@ -0,0 +1,48 @@
+"""
+    get_bounds(model::MOI.ModelLike, ::Type{T}, x::MOI.VariableIndex)
+
+Return a tuple `(lb, ub)` of type `Tuple{T, T}`, where `lb` and `ub` are lower
+ and upper bounds, respectively, imposed on `x` in `model`.
+"""
+function get_bounds(
```
Comment (blegat) [inline src/Utilities/variables.jl]: It should be added somewhere to `docs/src/apireference.md`

**Pull request merged**